django/api/services/spreadsheet_uploader_prep.py

```diff
@@ -258,15 +258,20 @@ def location_checker(df, *columns, **kwargs):
         unique_values = set(series)
 
         communities = set()
-        # populate communities by calling the bcngws API with the values:
-        get_placename_matches(values, 200, 1, communities)
+        for i in range(0, len(values), batch_size):
```

Comment:
You may want to define a new `unique_values_list = list(unique_values)` and use that in the loop instead of `values`, since `values` may contain duplicate items and that could lead to a greater than necessary number of queries.